src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -390,17 +400,27 @@ void icm426xxGyroInit(gyroDev_t *gyro)
     spiWriteReg(dev, ICM426XX_RA_GYRO_CONFIG0, (3 - INV_FSR_2000DPS) << 5 | (odrConfig & 0x0F));
     delay(15);
 
+#if defined(USE_ACC_SPI_IIM42653)
+    STATIC_ASSERT(INV_FSR_32G == 4, "INV_FSR_32G must be 4 to generate correct value");
+    // TODO: 4 minus 4 left shifted by 5.... is 0.. why is this here?
+    spiWriteReg(dev, ICM426XX_RA_ACCEL_CONFIG0, (4 - INV_FSR_32G) << 5 | (odrConfig & 0x0F));
```

Comment:
No reason not to right? I mean it does make it a little less likely that the accel will end up clipping.

Reply:
Is there anywhere else in the code base that might have the 16G limit hard coded? I took a brief look but didn't find anything

Reply:
AFAIK there is no limit. But it should be quite hard to get more than 16G (except in crash),
And you are trading resolution (and usually noise) for larger range.
So ... are there cases when 16 is not enough?

Reply:
@ledvinap have you looked at raw accelerometer noise in BF? Working with drones commercially our drone has multiple IMU and the 16g will end up saturating occasionally, while the 24g one rarely ever hits that limit. So it is possible, but normal vibration free craft shouldn't have it, but some larger noisier drones could very well hit that limit on occasion.

Reply:
Right now this will make no difference in BF as the AHRS code uses strongly filtered acc and only if it is near to 1g. So until we get some other code that can make use of the ACC then it doesn't really matter a whole lot.